webui/react/src/pages/ExperimentDetails/ExperimentConfiguration.tsx

```diff
@@ -14,6 +14,15 @@ interface Props {
 const MonacoEditor = React.lazy(() => import('components/MonacoEditor'));
 
 const ExperimentConfiguration: React.FC<Props> = ({ experiment }: Props) => {
+  let publicConfig = {};
+  if (experiment.configRaw) {
+    const {
```

Comment:
**suggestion:** could use a small comment here indicating that we are removing `registry_auth` from the config.

Reply:
👍